core/templates/dev/head/css/oppia.css

```diff
@@ -3441,13 +3441,12 @@ md-card.preview-conversation-skin-supplemental-card {
   border-radius: 0;
   color: #fff;
   font-family: "Capriola", "Roboto", Arial, sans-serif;
-  font-size: 18px;
+  font-size: 14px;
   margin-top: -20px;
-  padding: 10px 0;
+  padding: 10px;
   position: absolute;
   text-transform: uppercase;
   transform: translateX(-50%);
-  width: 360px;
```

Comment:
@anthkris Yes, there is another way around with respect to site design, by introducing `min-width:360px`. Also, the font-size decrement was an issue for me too. Could you please take a look at the mobile view with font-size being 18px and keeping `min-width:360px`. Font-size 14px seems to fit the both desktop and mobile view while translations switches. I guess we need many opinions on this. I am testing the translation of Dutch  (longest string compared to the rest )